README.rst

```diff
@@ -159,10 +159,9 @@ Rather, you can just import ``mayavi.mlab`` and pass it to any figure list that
 
 Installation for developers
 ---------------------------
-
-Once these are installed,
-to install from github, just ``git clone https://github.com/jmfranck/pyspecdata.git`` then move to the directory where setup.py lives,
-and do
+Typically, we create a folder on the hard drive called ``git`` or ``git_repos``, where you will put all of your code and the pyspecdata folder that you are about to clone. After installing the packages via the anaconda prompt, switch to your git bash terminal and move into your ``git`` folder.  
+To install from github, just ``git clone https://github.com/jmfranck/pyspecdata.git``. Then switch over to the anaconda prompt and move to the directory where setup.py lives (ex. ``git/pyspecdata/``),
```

Comment:
we decided it was best to just run setup.py on anaconda but git clone is a git bash command I'm pretty sure so we have to clone in git bash and then move back to anaconda to run setup.py right?

Reply:
The business about the directories is not necessary... People can organize their files however they like. On the other hand, I would explicitly say that it's best not to run setup.py inside the bash terminal.

Reply:
this comment has not been addressed